README.md

```diff
@@ -369,4 +370,242 @@ You may find it helpful to see the Kubernetes resources visually which can be ac
\ No newline at end of file
+After deploying the DBP, when accesing one of the applications, for example Process Services, if you receive the error message *We're sorry Invalid parameter: redirect_uri*, the `redirectUris` parameter provided for deployment is invalid. Make sure the `alfresco-infrastructure.alfresco-identity-service.client.alfresco.redirectUris` parameter has a valid value when installing the chart. For more details on how to configure it, check this [guide](https://github.com/Alfresco/alfresco-identity-service#changing-alfresco-client-redirecturis).
+
+
+# Docker for Desktop - Windows
+
+
+### 1. Install Docker for Desktop
+
+Check recommended version [here](https://github.com/Alfresco/alfresco-dbp-deployment/blob/master/README-prerequisite.md#docker-desktop).
+
+### 2. Enable Kubernetes
+
+In the 'Kubernetes' tab of the Docker settings,  click the 'Enable Kubernetes' checkbox.
+
+### 3. Increase Memory and CPUs
+
+In the Advanced tab of the Docker preferences, set 'CPUs' to 4.
+
+While Alfresco Digital Business Platform installs and runs with only 8 GiB allocated to Docker, 
+for better performance we recommend that 'Memory' value be set slightly higher, to at least 10 - 12 GiB
+(depending on the size of RAM in your workstation). 
+
+### 4. Change/Verify Context
+
+If you have previously deployed the DBP to AWS or minikube you will need to change/verify that the `docker-for-desktop` context is being used.
+
+```bash
+kubectl config current-context                 # Display the current context
+kubectl config use-context docker-for-desktop  # Set the default context if needed
+```
+
+### 5. Restart docker  
+
+Docker can be faulty on its first start. So, it is always safer to restart it before proceeding. Right click on the docker icon in the system tray, then left click "restart...". 
+
+### 6. Install helm
+
+Install chocolatery using command prompt (with admin rights). 
+
+```bash
+@"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
+```
+
+Initialize Helm Tiller (Server Component)
+
+```bash
+helm init
+```
+
+```bash
+choco install kubernetes-helm
+```
+
+### 7. Create your namespace
+
+```bash
+kubectl create namespace <yourNamespace>
+```
+
+### 8. Pull secrets
+
+Log in to quay.io
+```bash
+docker login quay.io
+```
+Give username and password if prompted.
+
+Generate a base64 value for your dockercfg, this will allow Kubernetes to access quay.io
+
+```bash
+certutil -encode "%USERPROFILE%\.docker\config.json" tmp.b64 && findstr /v /c:- tmp.b64
```

Comment:
I have now specified that it needs to be ran from the command prompt here:
![image](https://user-images.githubusercontent.com/42536841/53153470-3f0f6300-35b0-11e9-9e13-88736d27fa12.png)

I didn't manage to find a way to avoid creating tmp.b64 without making the user manually do the conversion such as by using https://www.base64encode.org/, so I instructed the user to then delete the file:

![image](https://user-images.githubusercontent.com/42536841/53153608-9f060980-35b0-11e9-8c90-774f3b9a0a3e.png)